en/guide/toolchain.md

```diff
@@ -13,164 +13,101 @@ Below we explain how to set up a minimal build setup (**CMakeLists.txt**) file f
+> **Note** The file format and required modifications are self-explanatory. 
+> If additional information is required see the [cmake documentation](https://cmake.org/cmake/help/latest/manual/cmake-commands.7.html).
 
-We'll explain some of it in the next section.
 
+### DroneCore Installed Locally {#dronecore_local_install}
 
-### DroneCore Installed Locally
+> **Tip** Where possible install Dronecore [system wide](../contributing/build.md#dronecore_system_wide_install) and follow the instructions in the [previous section](#dronecore_installed_system). 
 
-The **CMakeLists.txt** is more complicated when DroneCore is installed locally, because you need to specify where the build should find both headers and library files. 
+**CMakeLists.txt** is more complicated when DroneCore is [installed locally](../contributing/build.md#dronecore_local_install), because you need to specify where the build should find both headers and library files. The example file below assumes that DroneCore was installed locally into the directory **/install/**.
 
 The new lines are marked up using `# >>> ADDED` comments below.  You have to change the same information as before: *your_project_name*, *your_executable_name* and *your_source_file*. You may also need to change the location of the headers/library files, depending on where your application is hosted relative to the **DroneCore** project root directory (this example assumes that your application is nested two levels deep, just like our example code).
 
 ```cmake
 cmake_minimum_required(VERSION 2.8.12)
+
+## Specify your project's name
 project(your_project_name)
 
-# Add platform/compiler specific flags
+# Enable strict handling of warnings
 if(NOT MSVC)
     add_definitions("-std=c++11 -Wall -Wextra -Werror")
 else()
     add_definitions("-std=c++11 -WX -W2")
-    set(platform_libs "Ws2_32.lib")
 endif()
 
-# Add DEBUG define for Debug target
-set(CMAKE_CXX_FLAGS_DEBUG "-DDEBUG")
-
-# This finds thread libs on Linux, Mac, and Windows.
-find_package(Threads REQUIRED)
-
-# >>> ADDED
+# >>> 
 # Specify include directories
 include_directories(
     # Not needed if installed system-wide
     ${CMAKE_SOURCE_DIR}/../../install/include
 )
-# <<< 
+# >>>
 
-# Specify your binary name, and list of source files used to create it.
+# Specify your app's executable name, and list of source files used to create it.
 add_executable(your_executable_name
     your_source_file.cpp
+    # ... any other source files
 )
 
 # >>> ADDED
 # Specify variable 'dronecore_lib' containing location of DroneCore library.
 if(WINDOWS)
     set(dronecore_lib "${CMAKE_SOURCE_DIR}/../../install/lib/dronecore.lib")
 else()
-    set(dronecore_lib "${CMAKE_SOURCE_DIR}/../../install/lib/libdronecore.a")
+    set(dronecore_lib "${CMAKE_SOURCE_DIR}/../../install/lib/libdronecore.so")
```

Comment:
I have not yet tested this. 
- Is it correct to use .lib and .so as shown? Anything else that might need to change?

Reply:
I think that's correct but note that you can't use `if(WINDOWS)`. It should be `if(MSVC)`.